### PR TITLE
BERT quantization notebook

### PR DESCRIPTION
**Pull request by yufenglee**
**Description**: Describe your changes.

**Motivation and Context**
- Why is this change required? What problem does it solve?
- If it fixes an open issue, please link to the issue here.

### CODE REVIEW

File: onnxruntime/python/tools/quantization/notebooks/Bert-GLUE_OnnxRuntime_quantization.ipynb [hunk 1]
```diff
@@ -0,0 +1,1660 @@
...
+      "Requirement already satisfied: threadpoolctl>=2.0.0 in /home/yufeng/anaconda3/envs/pytorch/lib/python3.6/site-packages (from scikit-learn->sklearn) (2.1.0)\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Install or upgrade PyTorch 1.5.0 and OnnxRuntime 1.4.0 for CPU-only.\n",
+    "import sys\n",
+    "!{sys.executable} -m pip install --upgrade torch==1.5.0+cpu torchvision==0.6.0+cpu -f https://download.pytorch.org/whl/torch_stable.html\n",
+    "!{sys.executable} -m pip uninstall -y onnxruntime\n",
+    "!{sys.executable} -m pip install -i https://test.pypi.org/simple/ ort-nightly\n",
```
Comment (tianleiwu) [inline onnxruntime/python/tools/quantization/notebooks/Bert-GLUE_OnnxRuntime_quantization.ipynb:133]: How about installing official package when 1.4.0 is release.

File: onnxruntime/python/tools/quantization/notebooks/Bert-GLUE_OnnxRuntime_quantization.ipynb [hunk 2]
```diff
@@ -0,0 +1,1660 @@
...
+    "```\n",
+    "The last command will launch Jupyter Notebook and we can open this notebook in browser to continue."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 0.1 Install packages\n",
+    "Let install the nessasary packages firstly to start the tutorial.We will install PyTorch 1.5, OnnxRuntime 1.4,0, latest ONNX, OnnxRuntime-tools, transformers, and sklearn."
```
Comment (tianleiwu) [inline onnxruntime/python/tools/quantization/notebooks/Bert-GLUE_OnnxRuntime_quantization.ipynb:49]: Let's

File: onnxruntime/python/tools/quantization/notebooks/Bert-GLUE_OnnxRuntime_quantization.ipynb [hunk 3]
```diff
@@ -0,0 +1,1660 @@
...
+   "metadata": {},
+   "source": [
+    "# Bert Quantization with ONNX Runtime on CPU"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In this tutorial, we will load a fine tuned [HuggingFace BERT](https://huggingface.co/transformers/) model trained with [PyTorch](https://pytorch.org/) for [Microsoft Research Paraphrase Corpus (MRPC)](https://www.microsoft.com/en-us/download/details.aspx?id=52398) task , convert the model to ONNX, and then quantize PyTorch and ONNX model respectively. Finally, we will demonstrate the performance, accuracy and model size of the quantized PyTorhc and OnnxRuntime model in the [General Language Understanding Evaluation benchmark (GLUE)](https://gluebenchmark.com/)"
```
Comment (tianleiwu) [inline onnxruntime/python/tools/quantization/notebooks/Bert-GLUE_OnnxRuntime_quantization.ipynb:22]: typo: PyTorhc => PyTorch.
Could you run some spelling checking on the text?

File: onnxruntime/python/tools/quantization/notebooks/Bert-GLUE_OnnxRuntime_quantization.ipynb (hunk 1 above)
Comment (yufenglee) [reply onnxruntime/python/tools/quantization/notebooks/Bert-GLUE_OnnxRuntime_quantization.ipynb:133]: yeah, it is a placeholder

File: onnxruntime/python/tools/quantization/notebooks/Bert-GLUE_OnnxRuntime_quantization.ipynb [hunk 4]
```diff
@@ -0,0 +1,962 @@
...
+    "| ORT | 438 | 112 |\n",
+    "\n",
+    "### Accuracy\n",
+    "OnnxRuntime achieves a little bit better accuracy and F1 score, even though it has small model size.\n",
+    "\n",
+    "| Metrics | PyTorch Quantization | ORT Quantization |\n",
+    "| --- | --- | --- |\n",
+    "| Accuracy | 0.84559 | 0.85784 |\n",
+    "| F1 score | 0.89340 | 0.90203 |\n",
+    "| Acc and F1 | 0.86950 | 0.87994 |\n",
```
Comment (tianleiwu) [inline onnxruntime/python/tools/quantization/notebooks/Bert-GLUE_OnnxRuntime_quantization.ipynb:920]: Remove the row of Acc and F1? That metric is rarely used.

File: onnxruntime/python/tools/quantization/notebooks/Bert-GLUE_OnnxRuntime_quantization.ipynb [hunk 5]
```diff
@@ -0,0 +1,962 @@
...
+    "\n",
+    "### Performance\n",
+    "\n",
+    "The evaluation data set has 408 sample. Table below shows the performance on my machine with Intel(R) Xeon(R) E5-1650 v4@3.60GHz CPU. PyTorch quantization achieves ~1.33x speedup, and ORT quantization achieves ~1.34x speedup. And ORT quantization can achieve ~1.27x speedup, comparing with PyTorch quantization. \n",
+    "You can run the [benchmark.py](https://github.com/microsoft/onnxruntime/blob/master/onnxruntime/python/tools/transformers/benchmark.py) for comparison on more models.\n",
+    "\n",
+    "|Engine | Full Precision Latency(s) | Quantized(s) |\n",
+    "| --- | --- | --- |\n",
+    "| PyTorch | 32.9 | 24.8 |\n",
```
Comment (tianleiwu) [inline onnxruntime/python/tools/quantization/notebooks/Bert-GLUE_OnnxRuntime_quantization.ipynb:929]: I suggest append version to engine name. If you test different version, the number might be different.

File: onnxruntime/python/tools/quantization/notebooks/Bert-GLUE_OnnxRuntime_quantization.ipynb [hunk 6]
```diff
@@ -0,0 +1,962 @@
...
+   "source": [
+    "## 3 Summary\n",
+    "In this tutorial, we demonstrated how to quantize a fine-tuned BERT model for MPRC task on GLUE data set. Let's summarize the main metrics of quantization.\n",
+    "\n",
+    "### Model Size\n",
+    "PyTorch quantizes torch.nn.Linear modules only and reduce the model from 438 MB to 181 MB. OnnxRuntime quantizes not only Linear(MatMul), but also the embedding layer. It achieves almost the ideal model size reduction with quantization.\n",
+    "\n",
+    "| Engine | Full Precision(MB) | Quantized(MB) |\n",
+    "| --- | --- | --- |\n",
+    "| PyTorch | 438 | 181 |\n",
```
Comment (tianleiwu) [inline onnxruntime/python/tools/quantization/notebooks/Bert-GLUE_OnnxRuntime_quantization.ipynb:910]: Append version to engine name.

File: onnxruntime/python/tools/quantization/notebooks/Bert-GLUE_OnnxRuntime_quantization.ipynb [hunk 7]
```diff
@@ -0,0 +1,962 @@
...
+    "In this tutorial, we will load a fine tuned [HuggingFace BERT](https://huggingface.co/transformers/) model trained with [PyTorch](https://pytorch.org/) for [Microsoft Research Paraphrase Corpus (MRPC)](https://www.microsoft.com/en-us/download/details.aspx?id=52398) task , convert the model to ONNX, and then quantize PyTorch and ONNX model respectively. Finally, we will demonstrate the performance, accuracy and model size of the quantized PyTorch and OnnxRuntime model in the [General Language Understanding Evaluation benchmark (GLUE)](https://gluebenchmark.com/)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 0. Prerequisites ##\n",
+    "\n",
+    "If you have Jupyter Notebook, you can run this notebook directly with it. You may need to install or upgrade [PyTorch](https://pytorch.org/), [OnnxRuntime](https://microsoft.github.io/onnxruntime/), [transformer](https://huggingface.co/transformers/) and other required packages.\n",
```
Comment (tianleiwu) [inline onnxruntime/python/tools/quantization/notebooks/Bert-GLUE_OnnxRuntime_quantization.ipynb:31]: transformer => transformers
